contracts/vaults/RibbonThetaVault.sol

```diff
@@ -395,21 +354,18 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
 
         scheduledWithdrawals[msg.sender] = 0;
         queuedWithdrawShares = queuedWithdrawShares.sub(withdrawShares);
+        uint256 withdrawAmount = withdrawAmountWithShares(withdrawShares);
 
-        (uint256 amountAfterFee, uint256 feeAmount) =
-            withdrawAmountWithShares(withdrawShares);
-
-        emit Withdraw(msg.sender, amountAfterFee, withdrawShares, feeAmount);
-        emit ScheduledWithdrawCompleted(msg.sender, amountAfterFee);
+        emit Withdraw(msg.sender, withdrawAmount, withdrawShares);
 
```

Comment:
we need to add logic in withdraw later for checking whether transfer of shares to vault is needed. if they opt out of self-custody they wont have any shares.

Reply:
Noted, can I address this in a following PR?